cfg/cfg.go

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright (c) 2012-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package cfg
+
+import (
+	"flag"
+	"os"
+	"strconv"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// URL che-machine-exec api server url
+	URL string
+	// StaticPath path to serve static resources
+	StaticPath string
+	// UseBearerToken - flag to enable/disable using bearer token to have access to k8s api.
+	UseBearerToken bool
+)
+
+func init() {
+	defaultURLValue := ":4444"
+	urlEnvValue, isFound := os.LookupEnv("API_URL")
+	if isFound && len(urlEnvValue) > 0 {
+		defaultURLValue = urlEnvValue
+	}
+	flag.StringVar(&URL, "url", defaultURLValue, "Host:Port address.")
+
+	defaultStaticPath := ""
+	staticPathEnvValue, isFound := os.LookupEnv("STATIC_RESOURCES_PATH")
+	if isFound && len(staticPathEnvValue) > 0 {
+		defaultStaticPath = staticPathEnvValue
+	}
+	flag.StringVar(&StaticPath, "static", defaultStaticPath, "/home/user/frontend - absolute path to folder with static resources.")
+
+	defaultUseTokenValue := false
+	useTokenEnv := "USE_BEARER_TOKEN"
+	useTokenEnvValue, isFound := os.LookupEnv(useTokenEnv)
+	if isFound && len(useTokenEnvValue) > 0 {
+		if v, err := strconv.ParseBool(useTokenEnvValue); err == nil {
+			defaultUseTokenValue = v
+		} else {
+			logrus.Errorf("Invalid value '%s' for env varible key '%s'. Value should be boolean", useTokenEnvValue, useTokenEnv)
+		}
+	}
+	flag.BoolVar(&UseBearerToken, "use-bearer-token", defaultUseTokenValue, "to have access to the kubernetes api")
```

Comment:
Well, it's not really true. With SA token we also have access. For motivation here is to avoid user impersonation as I've described above. Please consider reformulating it.